app/src/main/res/layout/activity_main.xml

```diff
@@ -15,9 +15,60 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="Hello World!"
-            app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintLeft_toLeftOf="parent"
             app:layout_constraintRight_toRightOf="parent"
             app:layout_constraintTop_toTopOf="parent"/>
 
+    <TextView
+        android:id="@+id/departure_text"
+        android:textSize="25sp"
+        android:layout_margin="5sp"
```

Comment:
I notice that it's done in the provided label above, but generally you should use dp for layout related sizes, such as margins, rather than sp. sp are a special unit used for text in xml files for android development as they contain an additional scaling factor which allows the user to set their phone to have a larger (or smaller) than usual text size. Using sp for layout sizes could have strange side effects if the user changes the font size on their phone! Does that make sense?